File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] ans = new int[2*n];
        int j=n;
        for(int i=0; i<n; i++)
        {
            ans[i]=nums[i];
            ans[j]=nums[i];
            j++;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.py
class Solution:
    def findDuplicate(self, nums: List[int]) -> int:
        d= {}
        for i in nums:
            d[i] = d.get(i,0) + 1
        for j in d.keys():
            if d[j] > 1:
                return j
        return -1